src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(double a) {
+        radius = a;
+    }
```

Comment:
Constructor params should have informative names
```suggestion
    public Circle(double radius) {
        this.radius = radius;
    }
```